ClassDemo/ClassDemo.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,10 @@
+namespace ClassDemo.Tests;
+
+public class UnitTest1
+{
+    [Fact]

```

Comment:
```suggestion
    [Theory]

```
I think this should be a theory